dts/bindings/display/solomon,ssd1673fb.yaml

```diff
@@ -42,49 +42,49 @@ properties:
       generation: define
 
     gdv-a:
-      type: hex
+      type: int
```

Comment:
But the values are in [hex format](https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/reel_board/reel_board.dts#L131)

Reply:
how the values are specified doesn't impact things, in the dts they can be int/hex/oct/binary.  In the yaml 'int' just means its a number.